server/helpers/db.js

```diff
@@ -214,6 +218,10 @@ const checkUserByEmailExist = async email => {
   const response = await knex("users").where({ email });
   return response.length === 0 ? false : true;
 };
+const checkUserByIDExist = async user_id => {
+  const response = await knex("users").where({ user_id });
+  return response.length === 0 ? false : true;
+};
```

Comment:
Try something like this:
`
const checkUserByIDExist = async (email, userId) => {
    const response = await knex("users").where('email', email).andWhereNot('user_id', userId);
    return response.length === 0 ? false : true;
};
`

Reply:
Sorry, I have just seen this comment.